content/docs/user-guide/networking/settings.md

```diff
@@ -66,7 +65,7 @@ The following CVars control client behavior.
 
 
 ## Server settings
-The following CVars control server behavior.
+The following [console variables](http://localhost:1313/docs/user-guide/appendix/cvars/) control server behavior.
```

Comment:
broken link - need to remove "http://localhost:1313"

Reply:
Sigh, copy-n-pasta fail.